outputs.tf

```diff
@@ -38,8 +37,22 @@ output "backup_plan_version" {
 #####
 # AWS Backup Selection Outputs
 ####
-
 output "backup_selection_id" {
   description = "The identifier of the backup selection"
   value       = concat(aws_backup_selection.main[*].id, [""])[0]
 }
+
+#####
+# AWS Backup SNS Notification Outputs
+####
+output "backup_sns_topic_arn" {
+  description = "The Amazon Resource Name (ARN) that specifies the topic for a backup vault’s events"
+  value       = var.sns_topic_arn != null ? null : concat(aws_sns_topic.main[*].arn, [""])[0]
```

Comment:
this condition should be using var.create_sns_topic, also output should either the created topic or the one passed into the module.